css/style.css

```diff
@@ -0,0 +1,109 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
```

Comment:
dezza, a gente fez com rem ontem mas deixando o html com 62.5% pra arredondar todas as medidas do resto por 10. precisamos padronizar isso em todos os css ou vai dar ruim, anotando aqui já aproveitando pra registrar pra todo mundo!

então fica:
html {
font-size: 62.5%;
}

e aí tudo que for px a gente divide por 10 o valor, por exemplo: 5px = 0.5rem, 40px = 4rem, 200px = 20rem, etc.

Reply:
Vou modificar o meu